bundle-generation/generate-bundles.py

```diff
@@ -470,6 +472,45 @@ def injectHelmFlowControl(deployment):
         - name: NO_PROXY
           value: {{ .Values.hubconfig.proxyConfigs.NO_PROXY }}
 {{- end }}
+"""     
+        if sizes:
+            for sizDeployment in sizes["deployments"]:
+                if sizDeployment["name"] == deployx["metadata"]["name"]:
+                    for container in sizDeployment["containers"]:
+                        if line.strip() == "resources: REPLACE-" + container["name"]:
+                            lines[i] = """        resources:
+{{-  if eq .values.hubconfig.hubSize "small" }}
+          limits:
+            cpu: """ + container["small"]["limits"]["cpu"] + """
+            memory: """ + container["small"]["limits"]["memory"] + """
+          requests:
+            cpu: """ + container["small"]["requests"]["cpu"] + """
+            memory: """ + container["small"]["requests"]["memory"] + """
+{{- end }}
+{{ if eq .values.hubconfig.hubSize "medium" }}
+          limits:
+            cpu: """ + container["medium"]["limits"]["cpu"] + """
+            memory: """ + container["medium"]["limits"]["memory"] + """
+          requests:
+            cpu: """ + container["medium"]["requests"]["cpu"] + """
+            memory: """ + container["medium"]["requests"]["memory"] + """
+{{- end }}
+{{-  if eq .values.hubconfig.hubSize "large" }}
+    limits:
+        cpu: """ + container["large"]["limits"]["cpu"] + """
+        memory: """ + container["large"]["limits"]["memory"] + """
+    requests:
+        cpu: """ + container["large"]["requests"]["cpu"] + """
+        memory: """ + container["large"]["requests"]["memory"] + """
+{{- end }}
+{{ if eq .values.hubconfig.hubSize "extra-large" }}
+    limits:
+        cpu: """ + container["extra-large"]["limits"]["cpu"] + """
+        memory: """ + container["extra-large"]["limits"]["memory"] + """
+    requests:
+        cpu: """ + container["extra-large"]["requests"]["cpu"] + """
+        memory: """ + container["extra-large"]["requests"]["memory"] + """
+{{- end }}
```

Comment:
The `limits` and `requests` for these sizes are not aligned with the other ones, this will lead to issues when rendering the YAML.